tasks/initialise.yml

```diff
@@ -22,27 +22,78 @@
         PGSETUP_INITDB_OPTIONS: >-
           --encoding=UTF8 --locale=en_US.UTF-8 --auth-host=md5
 
-    - name: postgres | postgresql config file
-      template:
+    - name: postgres | Check for presence of "Modified by ome postgresql ansible role" in file
+      ansible.builtin.lineinfile:
+        path: "{{ postgresql_dist_confdir }}/postgresql.conf"
+        line: "#Modified by ome postgresql ansible role"
```

Comment:
This check is slightly concerning me in the context of deployment upgrades. It makes sense for idempotence and future versions of the role.

In the scenario of an existing PSQL database installed with a previous version of the role, this line will not be present and the `postgresql.conf` will be treated as the default configuration file installed by the package and modified by the templating below no?